WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -227,6 +233,18 @@ extension AddAttributeOptionsViewModel {
         addNewOption(name: option.name)
     }
 
+    /// Sets the current attribute name to a new name
+    ///
+    func setCurrentAttributeName(_ newName: String) {
+        state.currentAttributeName = newName
+    }
+
+    /// Gets the current attribute name
+    ///
+    func getCurrentAttributeName() -> String {
+        return state.currentAttributeName
+    }
+
```

Comment:
I think this could be modeled better as a computed property like `allowsRename` since we are not performing any transformation but just accessing a value!